src/HeaderBar.vala

```diff
@@ -20,7 +20,8 @@
 
 public class Mail.HeaderBar : Gtk.HeaderBar {
     public bool can_mark { get; set; }
-    public Gtk.SearchEntry search_entry { get; construct; }
+    public bool can_search { get; set; }
+    public Gtk.SearchEntry search_entry { get; set; }
```

Comment:
Another thing I've just spotted, it shouldn't be necessary to change the `construct` to `set` here.